### PR TITLE
Fill in Tabs content

### PR DESCRIPTION
**Pull request by taysea**
Preview: https://deploy-preview-680--keen-mayer-a86c8b.netlify.app/components/tabs
Designs: https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component
What this PR includes:
- [x] Guidance
- [x] Variants (with Icon, tab states) -- could be elaborated
- [x] Figma link/Docs link
The theme changes will be put on the NEXT branch of grommet-theme-hpe, but I'm waiting on the colors to be finalized from the design side. As of now, it would look like this with the theme changes in place:
Screen Shot 2020-04-20 at 3 23 25 PM

Need to be updated:
- [ ] NEXT branch of theme (pending color approval on design side)
- [x] Grommet designer (in progress, but pending addition of disabled prop -- PR here: https://github.com/grommet/grommet-designer/pull/63)

### CODE REVIEW

File: aries-site/src/pages/components/tabs.js [hunk 1]
```diff
@@ -17,19 +23,132 @@ const Tabs = () => (
...
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
         <SubsectionText>
           Tabs allow a user to access content while maintaining the existing
-          context. It consists of a container, or box, with tab controls to
-          expose the contents of the container.
+          context. The Tabs component serves as the outer wrapper and contains
```
Comment (britt6612) [inline aries-site/src/pages/components/tabs.js:41]: They can be used to rapidly navigate between different content while maintaining existing context

Adding it navigating between tabs may be helpful

**Comment by britt6612** [comment]
Screen Shot 2020-04-21 at 10 38 44 AM
one question should there be some sort of gap between the icons ?

File: aries-site/src/pages/components/tabs.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/pages/components/tabs.js]: Done. : )

**Comment by taysea** [comment]
> Screen Shot 2020-04-21 at 10 38 44 AM
> 
> one question should there be some sort of gap between the icons ?

Looks like this deploy isn't using the most recent grommet -- my Tab changes from yesterday should allow the labels and icons to show. Let me look into it.

**Comment by britt6612** [comment]
> > Screen Shot 2020-04-21 at 10 38 44 AM
> > one question should there be some sort of gap between the icons ?
> 
> Are you on the deploy or a local branch? On the deploy it looks like this. If you're on local, maybe you need to update your node_modules to get latest grommet stable.
> 
> Screen Shot 2020-04-21 at 9 41 53 AM

makes sense I was on local I will update on my side

File: aries-site/src/pages/components/tabs.js [hunk 2]
```diff
@@ -17,19 +23,132 @@ const Tabs = () => (
...
+          label such as "Settings" or "Notifications" helps the user easily
+          decide which tab contains the content they need. Longer labels may
+          clutter the layout and create a more effortful experience for the
+          user.
+        </SubsectionText>
+        <SubsectionText>
+          <Text weight="bold">
+            If using Tabs as a navigation structure, be sure to connect them to
+            the browser location.
+          </Text>
```
Comment (halocline) [inline aries-site/src/pages/components/tabs.js:72]:
```suggestion
          </Text>{' '}
```

File: aries-site/src/examples/components/tabs/TabsExample.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+} from 'grommet';
+
+export const TabsExample = () => {
+  const [index, setIndex] = React.useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+
+  return (
+    <Box>
+      <Tabs activeIndex={index} onActive={onActive} justify="start">
+        <Tab title="General" a11yTitle="Hi there">
```
Comment (halocline) [inline aries-site/src/examples/components/tabs/TabsExample.js:20]: Curious about what the "Hi there" is supposed to communicate / how it is aiming to enhance this example.

Comment (taysea) [reply aries-site/src/examples/components/tabs/TabsExample.js:20]: Whooops, let me remove that. Was a lingering test to make sure aria-labels worked.

**Pull request merged**